ui/app/components/proxy-tag.js

```diff
@@ -0,0 +1,8 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+  tagName: 'span',
+  classNames: ['tag', 'is-secondary', 'is-structure'],
+
+  'data-test-proxy-tag': true,
+});
```

Comment:
This is a very specific component that’s only shown in two places. I considered generalising it but it seemed premature. `is-structure` is a bit of an awkward class name, I’m open to suggestions 😬

Reply:
> is-structure is a bit of an awkward class name, I’m open to suggestions 😬

Hmm I dunno, what does 'is-structure' mean? (I know that sounds a bit of a silly question!) Like, what is the reason to add this other class?

Reply:
I added it because it’s still a `tag` but it follows the Structure style. I’m just not sure what to do haha 🤷‍♀

Reply:
Ah I getcha, so you have another type of tag? Oh yeah, just looked at bulma, you already have `tag` there! Cool right I follow now. I think `is-structure` is just as good as anything +1 🤷‍♂ 😄 , I suppose once you eventually migrate over completely that extra class would go.

Umm, I suppose the only other potential suggestion I'd have is - you could remove the `tag` class and make it `structure-tag` so you aren't mixing bulma and structure? Dunno if you like that more than what you already have tho.

Reply:
hmm, I do like that, thanks, but I’ll wait to hear from M regardless haha